clumper/clump.py

```diff
@@ -130,6 +130,72 @@ def read_jsonl(cls, path: str, n=None):
         except Exception:
             raise RuntimeError("Error occured during reading in JSONL file")
 
+    def write_json(self, path: str, n=None):
+        """
+        Writes to a json file.
+
+        Arguments:
+            path: filename
+            n: Number of rows to write out. Useful when writing large files. If `None`, all rows are written.
+
+        Usage:
+
+        ```python
+        from clumper import Clumper
+        clump_one = Clumper.read_json("tests/data/pokemon.json")
+        clump_one.write_jsonl("tests/data/pokemon_copy.json")
+
+        clump_two = Clumper.read_json("tests/data/pokemon_copy.json")
+        assert clump_two.collect() == clump_one.collect()
+        """
+        to_dump = self.collect()
+        if n is not None:
+            if n <= 0:
+                raise ValueError("Number of lines to write must be > 0.")
+            to_dump = self.head(n).collect()
+
+        try:
+            # Create a new file and open it for writing
+            with open(path, "w") as f:
+                json.dump(to_dump, f)
+        except Exception:
+            raise RuntimeError("Error occured during writing JSON file")
+
+    def write_jsonl(self, path: str, n=None):
+        """
+        Writes to a jsonl file.
+
+        Arguments:
+            path: filename
+            n: Number of rows to write out. Useful when writing large files. If `None`, all rows are written.
```

Comment:
> There's merit to that idea but it comes at a cost. A new user needs to know what `**kwargs` are possible. If that's not obvious then it might be better to just grab the two keyword arguments and put them in explicitly.

That makes sense.